code/game/machinery/computer/_computer.dm

```diff
@@ -30,10 +30,10 @@
 
 /obj/machinery/computer/update_overlays()
 	. = ..()
-	if(machine_stat & NOPOWER)
-		. += "[icon_keyboard]_off"
-		return
-	. += icon_keyboard
+	if(icon_keyboard)
+		if(machine_stat & NOPOWER)
+			return . += "[icon_keyboard]_off"
+		. += icon_keyboard
```

Comment:
Here too.

Reply:
An early return is not intended in the icon_keyboard if